mycroft/util/__init__.py

```diff
@@ -109,5 +111,21 @@ def connected(host="8.8.8.8", port=53, timeout=3):
             return False
 
 
+def create_signal(signalName):
```

Comment:
That is definitely possible, many places will however need to be looked over and maybe redesigned to work nicely. I mainly suggested the above as a quick drop in replacement keeping the same api and functionality without touching the disk.